GOFevaluation/utils.py

```diff
@@ -407,10 +426,17 @@ def plot_equiprobable_histogram(data_sample, bin_edges, order=None,
                 j += 1
             i += 1
 
-    fig = mpl.pyplot.gcf()
+    ax.set_xlim(xlim)
+    ax.set_ylim(ylim)
```

Comment:
this is a very nice solution for most use-cases but for `count_density` the result will then be kind of misleading.. The easiest solution to fix this is to raise an error in case plot_xlim and/or plot_ylim are passed together with this mode.. But it's not the cleanest solution. If you can think of something better it would be awesome. If not, I think this could work for now.